### DISCUSSION TITLE
"This page is taking too long to load" while trying to install application

### DISCUSSION THREAD

**Discussion by mbroton** [community/community#34751] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello!
I've encountered a problem while trying to connect DigitalOcean account with Github.
Request to:
`https://github.com/apps/digitalocean/installations/new?state=some_key`
every time returns response with status code 502 after ~10 seconds:
![gh502](https://user-images.githubusercontent.com/50829834/193349495-eae82697-b851-4cc3-9c4a-59a7841af603.png)
DigitalOcean support responded that this may be a Github issue.
Path on DigitalOcean's side is:
Apps -> Create App -> Select GitHub -> Manage Access -> The above error
I'm not sure if this is the right place to report this, but I couldn't find anything better.
I've tested it on two different devices, on different networks, but the result is the same.

**Comment by lisawagner** [comment]
Hello,

I'm having the same issue trying to connect Vercel to GitHub. Setting up a new Vercel account using GitHub works, but unfortunately when I try 'Import Git Repository' it times out.

**Reply by aurelia-mp** [reply]
Hi,
Same problem here, did you find a solution?

**Reply by adiguno** [reply]
+1

**Reply by Noshi96** [reply]
+1

**Reply by johncanero** [reply]
+1

**Reply by hideak** [reply]
+1

**Reply by shehrozkapoor** [reply]
+1

**Reply by marwanbukhori** [reply]
+1

**Reply by ozansozuozgit** [reply]
+1

**Reply by omar-jalal** [reply]
+1

**Reply by 0xAskar** [reply]
Remove yourself from the me50 repository or other organizations. me50 work for most

**Reply by MefDev** [reply]
+1

**Reply by tushgaurav** [reply]
+1, I am using Railway.app to host my Django project.

**Reply by uuuunniii** [reply]
+1

**Comment by mbroton** [comment]
I'm in contact with DigitalOcean support, they've passed my case to review by some engineer. I will make an update here, after I receive an answer.
My guess would be that it's GitHub's problem since I've got problem with DO, and you have problem with Vercel.

BTW. I don't know how to contact directly GitHub - is there no way or am I just searching badly? Error message says that I should contact them, but it's really hard to find some direct way.

**Reply by aurelia-mp** [reply]
Thanks for the update. I've been able to send a support ticket to GitHub through this page: [https://support.github.com/request](https://support.github.com/request) but no answer so far. I just tried to clone my repository into a new github account, just in case, and still have the same issue...

**Reply by lisawagner** [reply]
Thank you @mbroton for contacting DO support and thank you @aurelia-mp for the GitHub support link.

Trying to set up a new connect between Vercel and GitHub is still not working today.  I sent Vercel support a ticket, so hopefully we can find a workaround.

**Comment by lisawagner** [comment]
Hi Everyone,

Found a solution from GitHub Support that works for connecting with a new Vercel setup. Hopefully it will work for your use-cases too:

1) Go to:  Settings -> Organizations
2) Remove yourself as collaborator (if possible) for organizations with many repositories

**Reply by lutzcc1** [reply]
This worked for me. I had the same issue while trying to setup [Render](https://render.com/docs/github#troubleshooting). Thanks!

**Reply by artikov** [reply]
It worked! me50 was the problem...

**Reply by sh-cho** [reply]
me50 was the problem for me too. Thanks!

**Reply by sebastianlaclau** [reply]
same here!

**Reply by Noshi96** [reply]
Same thanks!

**Reply by cmdlinebeep** [reply]
Same, thank you!!

**Reply by Jacob1802** [reply]
if you remove yourself from me50 is all the progress lost?

**Comment by mbroton** [comment]
Thanks @lisawagner for sharing a solution.
I just received a response from GitHub with similar information.
In my case, organization `me50` is the problem. It's used by HarvardX's CS50 course.
Unfortunately I don't want to lose all my progress in this course, so I'm waiting for further instructions from GH team.

**Reply by adiguno** [reply]
This worked for me. Thanks!

**Reply by megil-delrey** [reply]
Did you leave me50? And if you did, did you lose your progress?

**Reply by mbroton** [reply]
@kelancelo I didn't, I'm still waiting for GitHub's support to respond on that matter.
As a workaround I'm using second account on which I forked a repository that I wanted to get access to via DigitalOcean.

**Reply by louie337** [reply]
Worked for me, also the `me50`, what are the odds?

**Comment by danielalejandroalanis** [comment]
Thanks @mbroton. Leaving me50 organization was the solution for me.

Regards,

**Comment by 3kh0** [comment]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, and turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.

*If this helped please mark this as the answer!* ❤️

**Reply by mbroton** [reply]
Thanks for sharing the tips!
It might probably help in some cases, but not in this specific case that this discussion is about.
The cause of the problem is already known, and it can be solved by removing yourself as a collaborator from problematic organizations.

**Comment by SMKH-PRO** [comment]
For me it solved after removing me50 repository

**Reply by crazould** [reply]
thanks!

**Reply by 0xAskar** [reply]
this worked for me as well! I was able to stay in the other organizations

**Comment by BigMoonTech** [comment]
Yup definitely removing me50 worked for me too. I just had this issue.

**Reply by Marxi7** [reply]
Do you lose everything related to me50 after that ? for example, if you're still following one of their course, will you lose everything ?

**Comment by benjaminangafua** [comment]
Just leave me50 to get everything done. Thanks

**Comment by hughlilly** [comment]
Had to leave code50 and me50 orgs in order to get Render blueprints to work. Doesn’t seem like an ideal solution. Guess I’ll just have to start CS50 all over again at some point.

**Comment by mbroton** [comment, marked as answer]
I received an email from GitHub support that this bug was just fixed.
Just tested it and I can confirm that it works without leaving `me50` repository :)

**Reply by Dkhack** [reply]
I received an email from GitHub support that this bug was just fixed.
Just tested it and I can confirm that it works without leaving `me50` repository :)
